src/docker-in-docker/devcontainer-feature.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "docker-in-docker",
-    "version": "1.0.6",
+    "version": "1.0.7",
```

Comment:
This is going to be 1.0.8 now 😅 .  It's on the list to automate bumping these with an action